services/templates/service.go

```diff
@@ -144,8 +170,10 @@ func (s service) GetTemplate(c context.Context, templateID int64, userID int64)
 		return nil, fmt.Errorf("get object: %w", err)
 	}
 
+	defer resp.Body.Close()
+
 	var html []byte
-	_, err = result.Body.Read(html)
+	_, err = resp.Body.Read(html)
```

Comment:
We should utilize `ioutil.ReadAll` here I believe:

```go
htmlBytes, err := ioutil.ReadAll(resp.Body)
```

Reply:
done